api/src/domain/updateRoadmapSidebarCards.ts

```diff
@@ -32,6 +32,19 @@ export const updateRoadmapSidebarCards: UpdateRoadmapSidebarCards = (userData: U
     }
   }
 
+  if (hasCompletedFormation) {
+    showCard("access-certifications-nudge");
+  } else {
+    hideCard("access-certifications-nudge");
+  }
+
+  const hasCompletedTaxes = userData.taskProgress["register-for-taxes"] === "COMPLETED";
+  if (hasCompletedTaxes) {
+    hideCard("access-certifications-nudge");
+  } else {
+    showCard("access-certifications-nudge");
+  }
+
```

Comment:
I think we should turn this into 1 if block, instead of two
Having 2 separate sets of logic for displaying the card could lead to confusion

The first if block could be  (pseudo code)
`if (hasCompletedFormation and !hasCompletedTaxes) 
  showCard(-cardname-) 
else
  hideCard(-cardname-)`